src/test/java/net/openhft/chronicle/wire/marshallable/FieldInfoTest.java

```diff
@@ -244,6 +244,12 @@ public static Collection<Object[]> combinations() {
                         "  bracketType: NONE,\n" +
                         "  parent: !type net.openhft.chronicle.wire.marshallable.ScalarValues\n" +
                         "}\n" +
+                        ", !FieldInfo {\n" +
+                        "  name: dynamicEnum,\n" +
+                        "  type: !type net.openhft.chronicle.wire.marshallable.TestDynamicEnum,\n" +
+                        "  bracketType: UNKNOWN,\n" +
+                        "  parent: !type net.openhft.chronicle.wire.marshallable.ScalarValues\n" +
+                        "}\n" +
```

Comment:
This should use VanillaFieldInfo, because DynamicEnumFieldInfo is only for use on *fields of* dynamic enums